docs/self-managed/identity/deployment/configuration-variables.md

```diff
@@ -27,6 +27,23 @@ As a Spring Boot application, Identity supports any standard
 | `KEYCLOAK_SETUP_CLIENT_ID`           | The client to use for authentication during setup of the provided Keycloak.         | admin-cli                                                                                                                                                                |
 | `KEYCLOAK_URL`                       | The URL of the Keycloak instance to use.                                            | http://localhost:18080/auth                                                                                                                                              |
 
+## Camunda Identity SDK configuration
+
+Below, find the names and values for the Identity SDK to ensure proper authentication and authorization with Identity and the Identity provider for all components.
```

Comment:
```suggestion
Below, find the names and values for the Identity SDK to ensure proper authentication and authorization with Identity and the Identity provider for Web Modeler and Optimize.
```

The overarching section is for Web Modeler, Optimize, Console. However, console for all I know does not embed the Identity SDK.